app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningDatasetSelectionFragment.kt

```diff
@@ -34,6 +36,13 @@ class LearningDatasetSelectionFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         val dsListFragment =
             childFragmentManager.findFragmentById(R.id.LearningDatasetSelection_dataset_list)
+        
+        if (savedInstanceState == null) {
+            childFragmentManager.commit {
```

Comment:
Could refactor as this is duplicated with DatasetsOverviewFragment

Reply:
I don't see how though tbh

Reply:
You could make a util function, but it's only 3 lines so don't bother